### DISCUSSION TITLE
Showcase Submissions/使用案例提交

### DISCUSSION THREAD

**Discussion by gqcn** [gogf/gf-site#59] [Show and tell]
**IMPORTANT**: you can submit your case to our showcase category by posting in this discussion. However we do not guarantee that your case will added and how soon. Please be patient.

# Add your case to the GoFrame showcase
🧑‍🎨 You create an aesome website or component using GoFrame and want to show it to the world?

🙏 Please add it to our case showcase - it only takes a few seconds! ⏱

✒️ The only thing you have to do ➡️ **give us your approval!**

⚠️ Please only submit **YOUR** cases.

## Submit your case
Just post a comment below in **this** GitHub discussion with some basic information:

* **Title**: <= 60 characters
* **Description**: ≤120 characters
* **URL**: live production deployment URL of your case
* **Source code URL**: only for open-source cases
* **Tags**: see below the list of available tags

You can also **just give us the case URL** and we'll figure out the rest, but providing the other pieces of info remains appreciated and ensure the correct info get displayed 😄.

We will take the case screenshot ourselves.

## After your submission
We'll get your case added as soon as possible.

Please just wait - we'll answer later to avoid creating too many notifications for everyone. You can still unsubscribe if the notification bothers you—we will ping you when replying.

We'll add valid submissions to our case showcase in one batch, when we have time to do so.

In case your submission is invalid, we will tell you.

## Site tags

The list of available case tags:

* ~**favorite**: Our favorite GoFrame cases that you must absolutely check out~ (This tag is maintained by us - doesn't matter if you include it, but we'll ignore it anyway)
* **opensource**: Associated to an Open-Sourced project.
* **product**: Associated to a commercial product.
* **frontui**: Associated frontend UI case built using GoFrame.
* **adminui**: Associated administration UI case built using GoFrame.
* **library**: An open-sourced library, for import usage.
* **micro**: It uses micro-service architecture.
* **cloudnative**: It uses cloud-native architecture.
* **ai**: Associated with AI technology.
* **tool**: Associated with system or develop tool case.
* **v1**: Built using GoFrame v1.
* **v2**: Built using GoFrame v2.

## Notes
* Help us keep the showcase up-to-date: tell us when a case has the wrong data.
* You can ask for an update of your case showcase data too!
* Technically, you could open a PR yourself and edit [src/data/users.tsx](https://github.com/gogf/gf-site/edit/main/src/data/users.tsx). However, apart from very small edits, we prefer to handle the PR ourselves for various reasons.
* We keep the right to edit your submission before publication.
* We'll keep this discussion clean over time: submissions will be marked as resolved, and later deleted.

<hr />

**重要说明**：您可以通过在此讨论中发布来将您的案例到我们的【案例展示】栏目中。但是，我们不保证您的案例会被添加以及何时被添加。请耐心等待。

# 将您的案例添加到 GoFrame 案例展示
🧑‍🎨 您是否使用 GoFrame 创建了一个超赞的网站或开源组件，并希望将其展示给全世界？

🙏 请将它添加到我们的案例展示中 - 这只需要几秒钟！⏱

✒️ 您需要做的唯一事情 ➡️ **在这里提交您的案例！**

⚠️ 请仅提交**您自己的**案例。

## 提交您的案例
只需在**此** GitHub 讨论下方发表评论，并提供以下基本信息（中英文均可）：

* **标题**：不超过 60 个字符
* **描述**：不超过 120 个字符
* **网址**：您案例的实际生产部署网址。
* **源码地址**：仅限开源案例。
* **标签**：请参考下方可用标签列表。

您也可以**只提供案例的网址**，我们会负责处理其余部分。当然，提供完整的信息更有利于我们快速和正确地添加并展示您的案例到栏目中 😄。

我们会自行对您的案例进行截图。

## 提交后的流程
我们会尽快将您的案例添加到展示中。

请耐心等待 - 为了避免创建过多通知，我们会稍后回复。如果通知打扰到您，可以取消订阅 - 但我们回复时会提醒您。

我们会在有空时将有效提交的案例一次性添加到展示中。

如果您的提交无效，我们会告知您。

## 网站标签

以下是可用的案例标签列表：

* ~**favorite**：我们喜欢并推荐给大家的案例~（此标签由我们维护——即使您包含它，我们也会忽略）
* **opensource**：开源项目案例，通常是开源站点或组件，可供大家参考的源码案例。
* **product**：带有商业授权案例，该案例可能同时具有源码和商业授权。
* **frontui**：带有前台UI的使用案例。
* **adminui**：带有管理后台UI的使用案例。
* **library***: Go源码组件，供import使用。
* **micro**: 该案例支持微服务开发架构。
* **cloudnative**: 该案例支持云原生开发架构。
* **ai**: 该案例与AI技术相关。
* **tool**: 系统或者开发工具相关。
* **v1**: 使用GoFrame v1版本构建。
* **v2**: 使用GoFrame v2版本构建。

## 注意事项
* 帮助我们保持展示内容的更新：如果某个案例信息错误，请告知我们。
* 您也可以请求更新您的案例展示信息！
* 技术上，您可以自己提交 PR 并编辑 [src/data/users.tsx](https://github.com/gogf/gf-site/edit/main/src/data/users.tsx)。不过，除了非常小的编辑外，我们更倾向于自行处理 PR，原因多种多样。
* 我们保留在发布前编辑您提交内容的权利。
* 我们会定期清理此讨论：提交的内容会被标记为已解决，并在稍后删除。

**Comment by gqcn** [comment]
Title: EasyGoAdmin
Description: 基于Golang、GoFrame、Vue、ElementUI、MySQL等技术栈开发平台框架，拥有完善的(RBAC)权限架构和基础核心管理模块，可以一键CRUD生成整个模块的全部代码，本框架为一站式系统框架开发平台，可以帮助开发者提升开发效率、降低研发成本，同时便于后期的系统维护升级。
URL: https://www.easygoadmin.vip/
Source code URL: [only for open-source cases](https://gitee.com/easygoadmin/EasyGoAdmin_GoFrame_EleVue)
Tags: opensource, adminui, product

**Comment by gqcn** [comment]
Title: EventMesh
Description: EventMesh is a new generation serverless event middleware for building distributed event-driven applications.
URL: https://eventmesh.apache.org/
Source code URL: https://github.com/apache/eventmesh
Tags: opensource, cloudnative

**Comment by lifeng2400** [comment]
标题：50CMSgo国产化开源后台，小程序后台
描述：gofram+pearadmin构建的后端开发框架，可以快速搭建政府国产化网站，企业网站，微信小程序商城，html代码解耦合，传统html+css+js 超级便于维护，可以跨平台部署在winserver，以及各种过国产操作系统。
网址：
源码地址：https://gitee.com/dexters/50CMSgo
标签：opensource  adminui

**Comment by anyanfei** [comment]
title: laghaim 龙腾混乱冒险官网+QQ机器人+自定义游戏签到
Description: 官网是由goframe框架的SSR，整合QQ机器人曾经的cq-http，现在已全面改造为QQNT钩子的WS服务，目前上线一年多无故障
WebUrl: https://www.laghaim.cn/
Source code URL: 暂不提供
Tags: 混乱冒险,laghaim,goframe,MMORPG game

**Reply by gqcn** [reply]
Your site is fantastic! However, we believe that it does not provide valuable references for others in the open-source community. Therefore, we are unable to showcase your case to the public. Thank you for your support!

**Comment by ghost** [comment]
Title: Goframe-vue-element-admin
Description: 基于goframe2.7和vue-element-admin搭建的后台框架
URL: https://gitee.com/lixianpei727/goframe-vue-element-admin
Source code [URL: https://github.com/apache/eventmesh](https://gitee.com/lixianpei727/goframe-vue-element-admin)
Tags:

**Comment by lusson-luo** [comment]
Title: HarmonyOS-groundhog-charging-system
Description： 基于goframe和vue-element-admin，鸿蒙4.0，微信小程序的充电系统平台
URL：https://github.com/cheinlu/HarmonyOS-groundhog-charging-system
Source code: https://github.com/cheinlu/HarmonyOS-groundhog-charging-system
Tags: opensource, frontui, adminui, cloudnative

**Comment by huagelong** [comment]
标题：devinggo 
描述：devinggo 是一款基于 Vue 和 Goframe v2 的全栈开发平台，集成 JWT 鉴权、动态路由、消息队列等功能，提供丰富场景模板，助您快速构建企业级应用，专注业务开发。
网址：http://106.14.10.255:8070/system
源码地址：https://github.com/huagelong/devinggo
标签：opensource，adminui，v2

**Comment by zcool321** [comment]
标题：gmanager
描述：基于GoFrame V2框架的后台管理系统。支持登录、认证、组织机构、用户帐号、角色权限、菜单、配置、操作日志等模块
网址：https://gmanager.jflyfox.com
源码地址：https://github.com/goflyfox/gmanager
标签：opensource，gmanager，v2

**Comment by zcool321** [comment]
标题：gtoken
描述：基于GoFrame框架的token插件，通过服务端验证方式实现token认证；已完全可以支撑线上token认证，通过Redis支持集群模式；使用简单，大家可以放心使用；
网址：https://github.com/goflyfox/gtoken
源码地址：https://github.com/goflyfox/gtoken
标签：opensource，gtoken，v2

**Comment by xgd16** [comment]
标题：ai-mcp
描述：基于 Go 的 MCP 服务端，支持 SSE、受控命令执行、SQL/查询工具、时间工具，集成 GoFrame/MySQL，适配 IDE/Agent 即用。
网址：https://github.com/xgd16/ai-mcp
源码地址：https://github.com/xgd16/ai-mcp
标签：opensource,v2,mcp-tool,ai

**Comment by huanglishi** [comment]
标题：GoframePro
描述：基于GoFrame V2框架和字节跳动ArcoDesign前端框架创建开发框架，为开发者快速开发项目。完整开发体系：Auth权限控制管理、数据字典、附件管理（集成本地、阿里云、腾讯云、七牛云的对象存储）、详细操作日志、完善配置管理（系统邮箱、应用配置、插件动态配）、开发工具（代码生成、代码插件市场一键安装和卸载）、支持单点登录等，框架有完整使用文档和GoFly社区全力支持，将推出使用视频教程，社区有众多开发者支持可以提供使用经验和丰富的市场插件支持。
网址：[https://goframepro.goflys.cn/webadmin](https://goframepro.goflys.cn/webadmin)
源码地址： [https://goflys.cn/goframepro](https://goflys.cn/goframepro)
标签：product，adminui，v2，micro，tool

#### 管理后台
<img width="3200" height="1662" alt="wechat_2025-10-19_102416_564" src="https://github.com/user-attachments/assets/d4c5b0db-fe19-48f7-8d68-a2c12698d20a" />

#### 社区支持
<img width="3200" height="1662" alt="wechat_2025-10-19_111153_217" src="https://github.com/user-attachments/assets/fe4c8fcf-72ff-4071-902d-92d9b10659fa" />

**Comment by z312193608** [comment]
Title: XYGo Admin

Description: 基于 GoFrame v2 + Vue3 的开源后台管理框架，内置前后端 CRUD 代码生成、RBAC 权限、插件扩展，支持 MySQL、PostgreSQL 与单文件部署。

URL: https://demo.xygoadmin.com

Source code URL: https://github.com/z312193608/xygo-admin

Tags: opensource, product, adminui, v2

**Comment by demozx** [comment]
标题：PasteLayer—跨端剪贴板同步软件（GoFrame v2）

描述：基于 Golang、GoFramev2、MySQL、Redis、ants 等技术栈开发的 macOS 剪贴板历史管理工具，提供账号体系（邮箱注册登录 / JWT 刷新旋转 / 密码重置）、剪贴板条目与分组的跨端 LWW 增量同步、图片 BLOB 流式传输、多设备管理与配额控制。

URL：https://www.pastelayer.com

源代码 URL：不开源

标签：frontui、tool、v2

**Comment by BryonGo** [comment]
# aicodcms —开源cms

## 亮点总结

- **标题**：aicodcms：插件化 开源 CMS，34 语言站群 + 双模板引擎
- **描述**：基于 GoFrame v2的 MIT 开源 CMS，支持插件化扩展、Casbin RBAC 权限、站群级多语言、文章自动翻译、静态生成与双模板引擎，适合全球内容站。
- **网址**：https://aicodcod.com
- **源码地址**：https://github.com/BryonGo/aicodcms

## 核心特性

- **插件化架构**：模块实现 `Plugin` 接口，`init()` 自动注册，零耦合，删目录即卸载。
- **站群级多语言**：文章级翻译 + `/zh-CN/`、`/en-US/` 路径路由 + 34 种语言声明 + Sitemap hreflang 自动生成。
- **双模板引擎**：开发用 gview（热刷新），生产用 templ（类型安全、零 IO）。
- **Casbin RBAC 权限**：PMS 作为系统骨架，所有模块共用统一权限体系。
- **静态页面生成**：`MakeHtml` 一键生成文章/列表/栏目/标签页，发布即可被爬虫收录。
- **自动翻译引擎**：保存文章触发 Baidu / DeepL 翻译，结果缓存，人工翻译优先。
- **通用订单模型**：一张订单表承载商品、游戏 SDK、文章付费，Snowflake UID + 原子余额扣减。
- **灵活内容模型**：单文章表经 `model_id` 支持文章 / 下载 / 电影等多类型。
- **CmsCtx 双层缓存**：L1 请求级 + L2 进程级，加速页面渲染。

## 技术栈

- 后端：Go 1.25+ · GoFrame v2 · MySQL 8.x / SQLite
- 前端：Vue 3 管理后台
- 协议：MIT License

## 相关链接

- 官网：https://aicodcod.com
- 源码：https://github.com/BryonGo/aicodcms
- 后台前端：https://github.com/BryonGo/aicodcms-vue
- 在线 Demo：https://demo.aicodcod.com/（账号 admin / admin888，每小时重置）
